test/data/kb_uploadmethods.import_fasta_as_assembly_from_staging.spec.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
Added this spec for testing, as the other ones present had multiple file inputs (fwd and rev reads files)